examples/react-native/app.json

```diff
@@ -25,7 +25,8 @@
       }
     },
     "web": {
-      "favicon": "./assets/favicon.png"
+      "favicon": "./assets/favicon.png",
+      "bundler": "metro"
```

Comment:
tried to make it work for web too, but it'd be more involved, so it's only iOS / Android now